From ec82e453593ecc46a159d5d79c31e67d6a9a324d Mon Sep 17 00:00:00 2001
From: Torge Matthies <openglfreak@googlemail.com>
Date: Mon, 25 Apr 2022 05:24:26 +0200
Subject: [PATCH 10/16] Revert "user32: Support sending RIM_TYPEMOUSE through
 __wine_send_input."

This reverts staging patch patches/user32-rawinput-mouse/0007-user32-Support-sending-RIM_TYPEMOUSE-through-__wine_.patch
---
 dlls/win32u/message.c | 6 ------
 server/queue.c        | 3 ---
 2 files changed, 9 deletions(-)

diff --git a/dlls/win32u/message.c b/dlls/win32u/message.c
index 11111111111..11111111111 100644
--- a/dlls/win32u/message.c
+++ b/dlls/win32u/message.c
@@ -2534,12 +2534,6 @@ NTSTATUS send_hardware_message( HWND hwnd, const INPUT *input, const RAWINPUT *r
                 req->input.hw.rawinput.type = rawinput->header.dwType;
                 switch (rawinput->header.dwType)
                 {
-                case RIM_TYPEMOUSE:
-                    req->input.hw.rawinput.mouse.x = rawinput->data.mouse.lLastX;
-                    req->input.hw.rawinput.mouse.y = rawinput->data.mouse.lLastY;
-                    req->input.hw.rawinput.mouse.data = rawinput->data.mouse.ulRawButtons;
-                    req->input.hw.lparam = rawinput->data.mouse.usFlags;
-                    break;
                 case RIM_TYPEHID:
                     req->input.hw.rawinput.hid.device = HandleToUlong( rawinput->header.hDevice );
                     req->input.hw.rawinput.hid.param = rawinput->header.wParam;
diff --git a/server/queue.c b/server/queue.c
index 11111111111..11111111111 100644
--- a/server/queue.c
+++ b/server/queue.c
@@ -2296,9 +2296,6 @@ static void queue_custom_hardware_message( struct desktop *desktop, user_handle_
         msg_data->size     = sizeof(*msg_data) + report_size;
         msg_data->rawinput = input->hw.rawinput;
 
-        if (input->hw.msg == WM_INPUT && input->hw.rawinput.type == RIM_TYPEMOUSE)
-            msg_data->flags = input->hw.lparam;
-
         enum_processes( queue_rawinput_message, &raw_msg );
 
         if (raw_msg.foreground) release_object( raw_msg.foreground );
-- 
2.36.1

